### PR TITLE
♻️ Uniformize Dockerfiles

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
- [x] make use of buildx caching
- [x] uniformize Dockerfile among services
- [x] fix access rights to /home/scu (in particular the base folder (/home/scu) were not set so that user *scu* could write to that folder. This was causing issues in the *resource-usage-tracker* and *dask-scheduler*
- [x] add ENV variables to pass to resources-usage-tracker service

## Related issue/s
- relates to https://github.com/ITISFoundation/osparc-issues/issues/675

### CODE REVIEW

**Review by GitHK — APPROVED** [review]
Nice

File: packages/postgres-database/docker/Dockerfile [hunk 1]
```diff
@@ -16,10 +16,8 @@ FROM base as build
 
 RUN apt-get update \
   && apt-get install -y --no-install-recommends \
-    build-essential \
-    git \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
```
[the remark this answers is not in the document]
Comment (sanderegg) [reply packages/postgres-database/docker/Dockerfile:22]: This is in the build part of the Dockerfile, which is disregarded anyway in the end since it is a multi-stage Dockerfile.
Also caching will not work if the cache files are deleted in the RUN clause.
It makes sense to clean the cache if the we are in the production stage where we will spare quite some space.

File: services/agent/Dockerfile [hunk 2]
```diff
@@ -118,6 +127,9 @@ ENV PYTHONOPTIMIZE=TRUE
 
 WORKDIR /home/scu
 
+# ensure home folder is read/writable for user scu
+RUN chown -R scu /home/scu
```
Comment (mguidon) [inline services/agent/Dockerfile:131]: Why is this necessary? Because the Dockerfile copies files in the scu home as root? You usually do `COPY --chown`
Also the entrypoing scripts usually mess around additionally

**Review by mguidon — APPROVED** [review]
LGTM. Just a question

File: services/agent/Dockerfile (hunk 2 above)
Comment (sanderegg) [reply services/agent/Dockerfile:131]: the entrypoint.sh looks for all files with user=scu and changes the user/right according to the mounted folders right, but it does not check the /home/scu folder access itself.

**Review by pcrespov — APPROVED** [review]

**Review by mrnicegyu11 — APPROVED** [review]
lgtm thanks

**Review by pcrespov — APPROVED** [review]
